
From: William Lee Irwin III <wli@holomorphy.com>

All in-tree references to remap_page_range() have been removed by prior
patches in the series.  This patch, intended to be applied after some waiting
period for people to adjust to the API change, notice __deprecated, etc., does
the final removal of remap_page_range() as a function symbol declared within
kernel headers and/or implemented in kernel sources.

Signed-off-by: Andrew Morton <akpm@osdl.org>
---

 25-akpm/include/linux/mm.h |    7 -------
 1 files changed, 7 deletions(-)

diff -puN include/linux/mm.h~for-mm-only-remove-remap_page_range-completely include/linux/mm.h
--- 25/include/linux/mm.h~for-mm-only-remove-remap_page_range-completely	2004-10-03 16:43:55.850840936 -0700
+++ 25-akpm/include/linux/mm.h	2004-10-03 16:43:55.854840328 -0700
@@ -775,13 +775,6 @@ extern struct page * follow_page(struct 
 int remap_pfn_range(struct vm_area_struct *, unsigned long,
 		unsigned long, unsigned long, pgprot_t);
 
-static inline __deprecated /* since 25 Sept 2004 -- wli */
-int remap_page_range(struct vm_area_struct *vma, unsigned long uvaddr,
-			unsigned long paddr, unsigned long size, pgprot_t prot)
-{
-	return remap_pfn_range(vma, uvaddr, paddr >> PAGE_SHIFT, size, prot);
-}
-
 #ifdef CONFIG_PROC_FS
 void __vm_stat_account(struct mm_struct *, unsigned long, struct file *, long);
 #else
_
